dist/s_string.ok

```diff
@@ -1187,6 +1187,7 @@ nd
 needkey
 needvalue
 negint
+neighbouring
```

Comment:
We generally use American English, and I think it's neighboring without the `u`?

Reply:
Yup, thanks! Updated in [7757cf8](https://github.com/wiredtiger/wiredtiger/pull/7687/commits/7757cf8b0411f56452f74ebfc68a718b2cc8a00d)